tests/functional/workflows/python_pip/test_packager.py

```diff
@@ -33,7 +33,7 @@ def _create_app_structure(tmpdir):
 
 @pytest.fixture
 def sdist_reader():
-    return SDistMetadataFetcher()
+    return SDistMetadataFetcher(python_exe=sys.executable)
```

Comment:
Can we add a comment that we are removing `sys.executable` from business logic but we are using it here for testing purposes?